src/main/java/com/airlines/dao/DBConnector.java

```diff
@@ -0,0 +1,29 @@
+package com.airlines.dao;
+
+import com.mysql.cj.jdbc.MysqlDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ResourceBundle;
+
+public class DBConnector {
+
+    private MysqlDataSource dataSource = new MysqlDataSource();
+
+    public DBConnector() {
+        ResourceBundle appResource = ResourceBundle.getBundle("application");
+        dataSource.setUser(appResource.getString("jdbcUsername"));
+        dataSource.setPassword(appResource.getString("jdbcPassword"));
+        dataSource.setUrl(appResource.getString("jdbcURL"));
+        dataSource.setDatabaseName("airlines");
+    }
+
+    public Connection getConnection() {
+
+        try {
+            return dataSource.getConnection();
+        } catch (SQLException e) {
+            throw new RuntimeException("Can not obtain database connection");
```

Comment:
It's a good practice to make own custom exception, for example in this case, it could be:
`throw new DatabaseConnectivityException("Database connection could not be obtained", e);`

This exception you could in new package called `exception`

Reply:
fixed in